omnibus/partybus/bin/partybus

```diff
@@ -140,18 +140,18 @@ end
 
 def do_upgrade(migration_state, migrations)
   pending_migrations = migrations.select{ |m| m > migration_state }.sort
-  log("Latest Migration Available: #{migrations.last}")
-  log("Migrations to Run: #{pending_migrations}")
+  log("Latest Database Migration Available: #{migrations.last}")
+  log("Database Migrations to Run: #{pending_migrations}")
 
   # run them through the DSL
   pending_migrations.each do |migration|
-    log("Current Migration Version: #{migration_state}")
+    log("Current Database Migration Version: #{migration_state}")
```

Comment:
should we be specific about this and say PostgreSQL?

Reply:
I updated all of these